### ISSUE TITLE
Adjust UV map of UILineTextureRenderer as it change size

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#151]
<sup>Issue created by Alexander Mrosk as [Bitbucket Issue #&#8203;151](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/151) on 2017.07.04 00:02.</sup>
I'v been using the UILineTextureRenderer for a project where the player can change the points position at will. As the line change size the Sprite stretches ruining the line's look, I would like to know if there is a way to make it tile or re map the uv as it change sizes, if not, is this feature possible or in the plans to be implemented, the lack of this make the UILineTextureRenderer kind of pointless

**Comment by kkostenkov** [comment]
I've checked the project and looks like the feature is not working. Am I missing something?

**Comment by SimonDarksideJ** [comment]
Can you elaborate @kkostenkov or log a new issue with the problem you are facing?

**Comment by kkostenkov** [comment]
I'm very sorry for the late answer and distracting you in general. After all I managed to figure it out myself.

To leave a note for someone like me in the future:
My goal was to create a line with tiling texture just like on the screenshots you've provided in the beginning of the conversation. Unfortunately I failed to find out how to do it with the package v2.3.2.  🙈 
So my problem was to make UILineRenderer draw lines using full UVs and not only 1 pixel of width from the source sprite.

Luckily I've discovered that the package has `SegmentType.Full` way of assigning UVs that's not accessible via Inspector controls of the component. So in the end I just had to modify one line of UILineRenderer to get correct UVs. 
`segments.Add(CreateLineSegment(start, end, SegmentType.Middle));` -> `segments.Add(CreateLineSegment(start, end, SegmentType.Full));`
![checker-animated](https://github.com/user-attachments/assets/1bb61eeb-f4a4-450a-94cd-168758f35bd1)

(Animation comes from a shader obviously)
